workflow-templates/generate-translation-files.yml

```diff
@@ -0,0 +1,13 @@
+# Run generate-translation-files
+
+name: Generate Translation Files
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Not every repo uses `master`. You need to get at the default branch. I'm not sure the best way to do this but this script gets at the default branch one way: https://github.com/openedx/.github/blob/master/workflow-templates/upgrade-python-requirements.yml and `${{ github.event.repository.default_branch }}` is a way of referencing the default branch.

Reply:
Can you post a test where that works?

Reply:
I'm still working on this. 
I've changed the branch to $default-branch.
But now one of the steps is not working when used in a reusable flow: https://github.com/Carlos-Muniz/credentials/actions/runs/3198268265/jobs/5222574593

Reply:
That link is about tokens not a default branch, so I'm a bit confused.

In the template update-python-requirements workflow, $default-branch would not work as a variable within an actual script. For each repo you need to update that variable to the actual default branch name. It is substituted automatically if you add the workflow via the Actions UI: https://github.com/openedx/tcril-engineering/actions - click New Workflow button, and see the Open edX defined template workflows to choose from. If you choose update python, notice the resulting workflow is added with `main` subbed in to `$default-branch`